src/test/java/core/basesyntax/service/CsvFileReaderServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exceptions.EmptyFileException;
+import core.basesyntax.exceptions.ExistFileException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.CsvFileReaderServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class CsvFileReaderServiceTest {
+    private static CsvFileReaderService reader;
+    private static final String INPUT_FILE = "src/test/resources/InputData.csv";
+    private static final String EMPTY_FILE = "src/test/resources/EmptyFile.csv";
+    private static final String WRONG_FILE = "src/test/resources/WrongFile.csv";
+
+    @BeforeClass
+    public static void beforeClass() {
+        reader = new CsvFileReaderServiceImpl();
+    }
+
+    @Test
+    public void readFromFile_emptyFile_notOk() {
+        Assertions.assertThrows(EmptyFileException.class,
+                () -> reader.readFromFile(EMPTY_FILE));
+    }
+
+    @Test
+    public void readFromFile_nullFile_notOk() {
+        Assertions.assertThrows(NullPointerException.class,
```

Comment:
It strange to expect NPE, we neet to add check to readFromFile method for null input, and throw some RuntimeException